chatterbot/conversation/statement.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, text, **kwargs):
         self.text = text
         self.in_response_to = kwargs.pop('in_response_to', [])
 
+        # Store tags information into statements
+        self.tags = kwargs.pop('tags', '')
```

Comment:
Shouldn't this be `self.tags = kwargs.pop('tags', [])` so that the default would be an empty list?